docs/configuration.md

```diff
@@ -63,5 +63,38 @@ See mutant's configuration file, [mutant.yml](/mutant.yml), for a complete examp
 #### `mutation_timeout`
 
 Specify the maximum time, in seconds, a mutation gets analysed.
-Past this time the mutation analysis gets terminated and the result is currently being
-reported as uncovered.
+
+```yml
+---
+# Control the maximum time per mutation spend on analysis.
+# Unit is in fractional seconds.
+#
+# Default absent.
+#
+# Absent value: No limit on per mutation analysis time.
+# Present value: Limit per mutation analysis time to specified value in seconds.
+mutation_timeout: 1.0
+```
+
+Use `timeout` setting under `coverage_criteria` in the config file to control
+if timeouts are allowed to cover mutations.
+
+#### `coverage_criteria`
+
+A configuration file only setting to control which criteria apply to determine mutation coverage.
+
+```yml
+---
+coverage_criteria:
```

Comment:
Is it possible to set this via CLI flag? Once we have per-subject / per file config then customizing it that way would be good but I am thinking of doing something like having the fork abort coverage off in my local config (so I can find and report bugs) but considering it as a kill in CI so that PRs are never gated by segfaults.

Reply:
Allowing to configure each subject separately is the next step. 

But: Its unlikely it makes sense in the CLI, simply because the CLI would become very long. My next initiative would be to support inheriting (prefix match) based per subject configuration in the code, and use the config file as input.

Than the overnext step to support it from in-line code comments.

Only after that I'll consider making CLI (per subject) support for these.

Reply:
Ok, yeah, I think that makes sense. Looking forward to it 👍